src/tools/pkcs11-tool.c

```diff
@@ -2640,6 +2640,26 @@ gen_key(CK_SLOT_ID slot, CK_SESSION_HANDLE session, CK_OBJECT_HANDLE *hSecretKey
 			FILL_ATTR(keyTemplate[n_attr], CKA_KEY_TYPE, &key_type, sizeof(key_type));
 			n_attr++;
 		}
+		else if (strncasecmp(type, "GENERIC:", strlen(":GENERIC")) == 0) {
```

Comment:
why `GENERIC:` and `:GENERIC` (not the colon).

Reply:
fixed in rebased squashed commit. pkcs11-tool.1.xml was updated that updates man page.

Reply:
fixed